bootstrap/components/processors/rtjoin-bolt-topology-component.json

```diff
@@ -0,0 +1,72 @@
+{
+  "type": "PROCESSOR",
+  "name": "Join",
+  "subType": "JOIN",
```

Comment:
"name" & "subType" should be "RT-Join" or something similar. We already use "JOIN" for join-bolt-topology-component.json
https://github.com/hortonworks/streamline/blob/master/bootstrap/components/processors/join-bolt-topology-component.json#L4